.github/workflows/ci.yml

```diff
@@ -24,19 +24,30 @@ jobs:
       if: matrix.config.name == 'Linux'
       run: |
         sudo apt-get install libboost-system-dev libboost-filesystem-dev libboost-program-options-dev
+        sudo apt-get install gcc-multilib
```

Comment:
Are you sure that this is required? These are usually only required if you want to compile 32-bit code on a 64-bit platform.

Reply:
I used to get an error "fatal error: bits/libc-header-start.h: No such file or directory”. This error was resolved after installing the gcc-multilib package.

Reply:
Interesting.. Can you please try with just "gcc"? It should already be there but maybe a recommended dependency is missing.

Reply:
Just to double check I removed this package and tried running it [https://github.com/AnoushkaU/etiss/runs/1537679511?check_suite_focus=true]. The error persists.

However, the error still persists with just gcc [https://github.com/AnoushkaU/etiss/runs/1537747870?check_suite_focus=true].

Please check: Linux: Test(Linux/Mac), Line 251 for both runs.

Reply:
I have an idea what could be the issue here.. Another student that was working on the LLVMJIT encountered something similar on his system. It seems like we are missing include directories for the LLVMJIT. Installing this package could cause the problem to disappear because this package puts additional headers onto the system. However, it is really unrelated and this workaround could cause wrong headers to be included.

Please add after this line: https://github.com/tum-ei-eda/etiss/blob/master/JITImpl/LLVM/src/LLVMJIT.cpp#L165

    args.push_back("-isystem/usr/include/x86_64-linux-gnu");

Reply:
Works now!